Azure-Samples/active-directory-aspnetcore-webapp-openidconnect-v2
# Removed cookie consent banner and updated copyright year

## Purpose

* Removes the consent banner and updates the copyright year to 2022.
* Consent banner contained dead link to a `Privacy` page that did not exist in the sample.

![image](https://user-images.githubusercontent.com/93682173/160459346-fd49a73d-a1a7-4117-bb6f-02c98f6c9176.png)


## Does this introduce a breaking change?

```
[ ] Yes
[X] No
```

## Pull Request Type
What kind of change does this Pull Request introduce?


```
[ ] Bugfix
[ ] Feature
[ ] Code style update (formatting, local variables)
[ ] Refactoring (no functional changes, no api changes)
[ ] Documentation content changes
[X] Other... **UI update**

```

## How to Test
*  Get the code

```
git clone [repo-address]
cd [repo-name]
git checkout [branch-name]
```

Configure sample app according to readme and run

```
dotnet run
```

* Test the code. Consent banner should no longer be present and copy right year updated otherwise no other changes and function should be the same.

## What to Check
Verify that the following are valid
* Consent banner does not appear
* Copyright year updated to 2022

## Other Information
